tonic-build/src/prost.rs

```diff
@@ -15,6 +15,11 @@ use std::{
 ///
 /// Use [`compile_protos`] instead if you don't need to tweak anything.
 pub fn configure() -> Builder {
+    let type_attributes = vec![(
+        ".".to_string(),
+        "#[allow(clippy::derive_partial_eq_without_eq)]".to_string(),
+    )];
```

Comment:
As adding attributes might be only be configured via prost api, it seems like we need to set this as the default. However I'm not sure that this is an appropriate approach.

Reply:
Ugh I wonder if we should just fix this in prost then? Somewhere in this macro impl https://github.com/tokio-rs/prost/blob/master/prost-derive/src/lib.rs#L21 since really the problem is I don't want code generated stuff to impl Eq or we can put this in prost-build?

Reply:
I published https://github.com/tokio-rs/prost/pull/775 as to try solving this.